util/doc_ock.py

```diff
@@ -22,15 +24,31 @@ def register_user(user: models.User):
 
 
 def post_event(user: models.User, event: events.Event):
-    daemon.run(__post_thread, user, event)
+    caller_dict = __get_caller_data()
+    daemon.run(__post_thread, user, event, caller_dict)
 
-def __post_thread(user: models.User, event: events.Event):
+
+def __post_thread(
+    user: models.User, event: events.Event, caller_dict: Dict[str, Union[str, int]]
+):
     try:
         url = f"{BASE_URI}/track/{user.id}/{event.event_name()}"
         event.IsManaged = config_service.get_config_value("is_managed")
         add_user_activity_entry(
-            str(user.id), {"eventName": event.event_name(), **event.__dict__}
+            str(user.id),
+            {"eventName": event.event_name(), **event.__dict__},
+            caller_dict,
         )
         service_requests.post_call_or_raise(url, event.__dict__)
     except Exception:
         print(traceback.format_exc(), flush=True)
+
+
+def __get_caller_data() -> Dict[str, Union[str, int]]:
+    # not this, not the parent (post_event) but the one before
```

Comment:
not sure what this comment explains to me, what is meant with not this

Reply:
we talked about this the last time. Its in regards to the stack trace index to explain the number used. 
but I can remove it

Reply:
changed the comment